src/components/hooks/gtm-data-hooks.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react'
+import { getClientInformation, getDomain, getLanguage, isBrowser } from 'common/utility'
+
+const useGTMData = () => {
+    const [gtm_data, setGTMData] = React.useState(null)
+    const domain = getDomain()
+    const is_logged_in = getClientInformation(domain)
+    const has_dataLayer = isBrowser() && window.dataLayer
+
+    useEffect(() => {
+        if (gtm_data) {
+            has_dataLayer &&
+                window.dataLayer.push({
+                    ...gtm_data,
+                    loggedIn: is_logged_in,
```

Comment:
client information is an object, please pass in boolean here, and also please add
- visitorId , which is client_information.loginid
- currency , which is client_information.currency